node-manager/src/error.rs

```diff
@@ -284,11 +284,7 @@ impl From<bitcoin::util::address::Error> for MutinyJsError {
 
 impl From<PaymentError> for MutinyJsError {
     fn from(e: PaymentError) -> Self {
-        match e {
-            PaymentError::Invoice(_) => Self::InvoiceInvalid,
-            PaymentError::Routing(_) => Self::RoutingFailed,
-            PaymentError::Sending(_) => Self::RoutingFailed,
-        }
+        MutinyError::from(e).into()
```

Comment:
Ah, convert to `MutinyError` and then convert to `MutinyJsError`, yeah that makes sense, nice catch.